source/endpoints/jobs.rst

```diff
@@ -0,0 +1,153 @@
+404                    Job Not Found
+====================== ===============================================================
+
+
+``POST /jobs``
+*****************
+
+Submits a transcription job
+
+**CURL Examples**
+
+.. code:: javascript
+
+    curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: application/json" -d "{\"media_url\":\"https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3\",\"metadata\":\"This is a sample submit jobs option\"}"
+
+.. code:: javascript
+
+  curl -X POST "https://api.rev.ai/revspeech/v1beta/jobs" -H "Authorization: Bearer <api_key>" -H "Content-Type: multipart/form-data" -F "media=@media_file.mp3" -F "options={"metadata":"This is a sample submit jobs option for multipart"}"
+
+**Request Body (required)**
+
+====================== ===============================================================
+Content-Type           Example
+====================== ===============================================================
+application/json       Revspeech API Options
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                            "media_url": "https://support.rev.com/hc/en-us/article_attachments/200043975/FTC_Sample_1_-_Single.mp3",
+                            "metadata": "This is a sample submit jobs option",
+                            "callback_url": "https://www.example.com/callback"
+                        }     
+---------------------- ---------------------------------------------------------------
+multipart/form         ``Example Value``
+
+                       .. code:: javascript
+
+                         
+                         Key      Value
+                         -------  --------------------------
+                         media    any media file with audio
+                         options  options model (either as a string or json file)
+====================== ===============================================================
+
+**Responses**
+
+====================== ===============================================================
+Code                   Description
+====================== ===============================================================
+200                    Revspeech API Job
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "id": "111111",
+                          "status": "in_progress",
+                          "created_on": "2018-05-05T23:23:22.29Z"
+                        }       
+---------------------- ---------------------------------------------------------------
+400                    Bad Request
+
+                       ``Example Value``
+
+                       .. code:: javascript
+
+                        {
+                          "parameter": {
```

Comment:
I dont think this sample is quite right. it should be "parameters". and also type / title are peer to parameters, not nested in it. e.g. should match https://www.temi.com/api/reference/v1#error-messages-structure